src/main/scala/com/gilcloud/sbt/gitlab/GitlabPlugin.scala

```diff
@@ -107,8 +111,34 @@ object GitlabPlugin extends AutoPlugin {
         .value,
       publish := publish.dependsOn(headerAuthHandler).value,
       publishTo := (ThisProject / publishTo).value.orElse {
-        gitlabProjectId.value.map(p => "gitlab-maven" at s"https://${gitlabDomain.value}/api/v4/projects/$p/packages/maven") orElse
-        gitlabGroupId.value.map(g => "gitlab-maven" at s"https://${gitlabDomain.value}/api/v4/groups/$g/-/packages/maven")
+        gitlabProjectId.value.map(gitlabProjectRepository(gitlabDomain.value, _)) orElse
+        gitlabGroupId.value.map(gitlabGroupRepository(gitlabDomain.value, _))
+      },
+      resolvers ++= gitlabProjectId.value
+        .map(gitlabProjectRepository(gitlabDomain.value, _)) orElse
+        gitlabGroupId.value.map(gitlabGroupRepository(gitlabDomain.value, _)),
+      csrConfiguration := {
```

Comment:
You also have to do this for `updateClassifiers / csrConfiguration` and `updateSbtClassifiers / csrConfiguration`, otherwise the downloading of sources won't work

Reply:
@markvandertol `updateClassifiers` and `updateSbtClassifiers` [already depend](https://github.com/gilcloud/sbt-gitlab/blob/f63374b0d2a63cde06bd46fc0b4778039f36b051/src/main/scala/com/gilcloud/sbt/gitlab/GitlabPlugin.scala#L108-L109) on `headerAuthHandler`. Are you sure it is necessary? What's the easiest way to check?

Reply:
Running `updateClassifiers` turns out to be a pretty efficient way to check 😄